src/components/Comments/Comment.js

```diff
@@ -121,12 +121,12 @@ const Comment = ({
 
   const voteStyle = () => {
     if (totalVote > 0) {
-      return '#00ADB5';
+      return COLORS.holytosca;
     }
     if (totalVote < 0) {
-      return '#FF2E63';
+      return COLORS.red;
     }
-    return '#C4C4C4';
+    return COLORS.gray1;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color values in the `voteStyle` function have been replaced with dynamic colors from the theme object. This improves consistency and maintainability of the code.
```diff
-      return '#00ADB5';
+      return COLORS.holytosca;
...
-      return '#FF2E63';
+      return COLORS.red;
...
-    return '#C4C4C4';
+    return COLORS.gray1;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->